manim/mobject/mobject.py

```diff
@@ -2160,6 +2166,7 @@ def length_over_dim(self, dim: int) -> float:
             dim,
         ) - self.reduce_across_dimension(min, dim)
 
+    @internal
```

Comment:
why?

Reply:
I'm not particularly for or against it, it just didn't feel useful ig. On second thought, I think this one might be better to make public.